packages/xstate-angular/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@xstate/angular",
+  "version": "0.1.0",
+  "description": "XState tools for Angular",
+  "keywords": [
+    "state",
+    "machine",
+    "statechart",
+    "scxml",
+    "state",
+    "graph",
+    "angular",
+    "inject"
+  ],
+  "author": "Tomasz Ducin <tomasz.ducin@gmail.com>",
+  "homepage": "https://github.com/statelyai/xstate/tree/main/packages/xstate-angular#readme",
+  "license": "MIT",
+  "main": "dist/xstate-angular.cjs.js",
+  "module": "dist/xstate-angular.esm.js",
+  "type": "module",
```

Comment:
is this a requirement? It certainly makes the current `exports` configuration incorrect since now `.js` files will be treated as modules but they are actually scripts

Reply:
If possible - I would prefer removing this field from here.